Segment-ComScore/Classes/SEGComScoreIntegration.m

```diff
@@ -34,20 +34,24 @@ - (instancetype)initWithSettings:(NSDictionary *)settings andComScore:(id)scorAn
         SCORPublisherConfiguration *config = [SCORPublisherConfiguration publisherConfigurationWithBuilderBlock:^(SCORPublisherConfigurationBuilder *builder) {
             // publisherId is also known as c2 value
             builder.publisherId = settings[@"c2"];
-            builder.publisherSecret = settings[@"publisherSecret"];
-            builder.applicationName = settings[@"appName:"];
-            builder.usagePropertiesAutoUpdateInterval = [settings[@"autoUpdateInterval"] integerValue];
-            builder.secureTransmission = [(NSNumber *)[self.settings objectForKey:@"useHTTPS"] boolValue];
-            builder.liveTransmissionMode = SCORLiveTransmissionModeLan;
-
-            if ([(NSNumber *)[self.settings objectForKey:@"autoUpdate"] boolValue] && [(NSNumber *)[self.settings objectForKey:@"foregroundOnly"] boolValue]) {
-                builder.usagePropertiesAutoUpdateMode = SCORUsagePropertiesAutoUpdateModeForegroundOnly;
-            } else if ([(NSNumber *)[self.settings objectForKey:@"autoUpdate"] boolValue]) {
-                builder.usagePropertiesAutoUpdateMode = SCORUsagePropertiesAutoUpdateModeForegroundAndBackground;
-            } else {
-                builder.usagePropertiesAutoUpdateMode = SCORUsagePropertiesAutoUpdateModeDisabled;
-            }
+            builder.secureTransmissionEnabled = [(NSNumber *)[self.settings objectForKey:@"useHTTPS"] boolValue];
         }];
+        
+        SCORAnalytics.configuration.applicationName = settings[@"appName:"];
+        
+        NSInteger usageConfigurationProperty = SCORUsagePropertiesAutoUpdateModeDisabled;
+        
+        if ([(NSNumber *)[self.settings objectForKey:@"autoUpdate"] boolValue] && [(NSNumber *)[self.settings objectForKey:@"foregroundOnly"] boolValue]) {
+            usageConfigurationProperty = SCORUsagePropertiesAutoUpdateModeForegroundOnly;
+        } else if ([(NSNumber *)[self.settings objectForKey:@"autoUpdate"] boolValue]) {
+            usageConfigurationProperty = SCORUsagePropertiesAutoUpdateModeForegroundOnly;
+        }
+        SCORAnalytics.configuration.usagePropertiesAutoUpdateMode = usageConfigurationProperty;
+        
+        SCORAnalytics.configuration.usagePropertiesAutoUpdateInterval = [settings[@"autoUpdateInterval"] intValue];
```

Comment:
Any particular reason we changed this from `[settings[@"autoUpdateInterval"] integerValue];` to `[settings[@"autoUpdateInterval"] intValue];`?

Reply:
It was a typo. IntegerValue is the correct usage.

Reply:
Yes, because usagePropertiesAutoUpdateInterval is now an int, and so it was losing precision.